### ISSUE TITLE
python3-mb: use disambiguity to create unique directory names

### ISSUE THREAD

**Issue by pimzand** [jack-cli-cd-ripper/jack#28]
Some artists have released multiple albums with the same name, like
https://musicbrainz.org/artist/8e66ea2b-b57b-47d9-8df0-df4630aeb8e5

Jack should be able to automatically create unique directory name for these releases, possibly using the disambiguity information in the MusicBrainz data.

**Comment by pimzand** [comment]
There's an `--add-disambiguation` option now. It will add disambiguation to the album title,
provided there is disambiguation info in the MusicBrainz metadata.
Useful if you have Peter Gabriel's first four albums in your collection, which are all called the same.
Using this option, the directory names will be called
```
Peter Gabriel (Car)
Peter Gabriel (Scratch)
Peter Gabriel (Melt)
Peter Gabriel (Security)
```
We might want the same for similar named artists, although there are less examples for that,
and there's a less of a chance that this will result in conflicting directory names.

What _is_ needed, is the ability for jack to remember that `--add-disambiguation` has been used,
so it could be used again for a next `--rename` run automatically,

I have run into this after a mass refresh of all musicbrainz metadata.

Jack could store the need to disambiguate in either jack.musicbrainz or jack.progress.

**Comment by pimzand** [comment]
Fixed in c8cf41339b41e7308497892b86217fd52d8a8250

Jack will now save the add-disambiguation status in jack.musicbrainz, when `--add-disambiguation` and `--query-now` are used at the same time. Use --query-now --refresh-metadata to forget about it.
